### PR TITLE
Brazil

### PR DESCRIPTION
**Pull request by JB-Cezar**
hi guys please check if my code is correct before merging

### CODE REVIEW

File: scss/devices/brasil/_desktop.scss [hunk 1]
```diff
@@ -0,0 +1,153 @@
+@media (min-width: $desktopVersion){
+    // NAV START
+    details {
+        display: none;
+    }
+    nav {
+        height: 10vh;
+        padding: 0 3%;
+        align-items: center;
+        position: sticky;
```
Comment (Dayhun) [inline scss/devices/brasil/_desktop.scss:11]: z-index:5;
I also added this in home and korea page nav.
Cuz in my page, the pictures in the main is over the nav.
You can add if you need.

File: scss/devices/brasil/_desktop.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            color: black;
+            font-weight: 600;
+            &:hover {
+                color: $nav;
+            }
+        }
+    }
+    // NAV END
+    // HEADER START
+    .kHeader {
```
Comment (Dayhun) [inline scss/devices/brasil/_desktop.scss]: I think you should write bHeader. Isnt it?

**Review by Dayhun — COMMENTED** [review]
Check my review Please! Thanks.

File: scss/devices/brasil/_desktop.scss (hunk 2 above)
Comment (JB-Cezar) [reply scss/devices/brasil/_desktop.scss]: actually, as it is a class, its name doesn't really matter, it will end up working anyway

**Pull request merged**